src/fondant/pipeline/lightweight_component.py

```diff
@@ -121,6 +172,32 @@ class PythonComponentOp(cls, PythonComponent):
             def image(cls) -> Image:
                 return image
 
+            @classmethod
```

Comment:
You can make this a class property by combining the `classmethod` and `property` decorators.

Reply:
hmm it doesn't seem correct, should I apply `getters` and `setters`

![image](https://github.com/ml6team/fondant/assets/47530815/5dba354e-4246-47e2-a243-13ddcd387aef)

Reply:
It should be reversed, but seems like it only works for Python 3.9 and 3.10 ([docs](https://docs.python.org/3.11/library/functions.html#classmethod)).

Just making it a class attribute could work as well:
```python
class BaseClass:
    consumes: 

class Class(BaseClass):
    consumes=consumes_  # cannot be the same name
```

Reply:
hmm I still don't quite follow what should be done here, aren't the class methods needed for the decorators? what's the need for attributes in this example?